frontend/lib/ui/style/layout.css

```diff
@@ -90,4 +90,19 @@ article#polling-station-choice-form {
 nav#progress {
   width: 26.5rem;
   padding: 2.5rem 5rem;
+  gap: 2.5rem;
```

Comment:
Is it possible to move this styling to a module/component css?

Reply:
Done in c3454c192dbc8c694a1c85fce5b8ed5ee45d7605 and 5d75e8a50a0b014b71f44026094f1f0f86b585bd